metaflow/__init__.py

```diff
@@ -143,6 +143,9 @@ class and related decorators.
     DataArtifact,
 )
 
+# Data class needed for Client.
+from .tuple_util import ForeachFrame
```

Comment:
nit, you probably don't need to import ForeachFrame. You can probably simply do: `from . import tuple_util`

Reply:
done